.gitignore

```diff
@@ -13,6 +13,8 @@ genesis.ssz
 
 # IntelliJ
 /*.iml
+<<<<<<< HEAD
```

Comment:
```suggestion
```

Reply:
merged without noticing but removed in the 4844 branch, ty @realbigsean !